### ISSUE TITLE
Hiding/Showing portals for players based on årivate, friends, guilds?

### ISSUE THREAD

**Issue by Sketaful** [SpikeHimself/XPortal#95]
I love how easy and well XPortal works. The problem is that it kinda remove any chance of having portals hidden for those special places you like  to be alone no matter if it's a giant server or just some friends playing. 

I would love if, first and all, it was possible to somehow mark a portal as private making it only show up in your own list. If other players check portal destinations they shouldn't see it. It's the main thing I'd want to be able to do. 

As an after thought, since I play around with some more modded server I'd like to see the private portals be expanded to support friends and guilds as choices to be shown to. The ability to only show your portals for guild mates and so on like support for ex. Smoothbrains Guilds and/or Groups mods. ;)

**Comment by Jester2020** [comment]
I came to ask for the same thing. The simple version just being a flag to make a portal private to hide ti from the global list so other players cant see it so a player can play solo in the world or share their portals co-op. An additional option would be a locked flag so only the player that created the portal could use it.

**Comment by ZenDragonX** [comment]
This is a good idea!  yes please.

**Comment by Sketaful** [comment]
Smooth rain's "Target portal" let you do this so it's possible. I'd prefer using portal than clicking the map though.

**Comment by Jester2020** [comment]
> Smooth rain's "Target portal" let you do this so it's possible. I'd prefer using xportal than clicking the map though.

yeah i tried TargetPortal by SmoothBrain. I personally don't care for it for a few reasons. I'm sure others love it.  Just personal preference.

**Comment by Vapok** [comment]
All of this request feature has been included in my Merge Request #138
